platzky_b4uleave/entrypoint.py

```
@@ -0,0 +1,118 @@
+from flask import Flask, Response
```

Ruff (0.8.2)

`flask.Flask` imported but unused

Remove unused import: `flask.Flask`

(F401)